focus-areas/what/HardwareDiversity.txt

```diff
@@ -0,0 +1,37 @@
+﻿Hardware Diversity
+
+
+Question: What hardware are people using to develop with.
+
+
+Description:
+Determine what affect hardware has on development.  
+
+
+Objectives:
+Determine what effects hardware has on development.  Determine if hardware has an effect on frequency of development.  Determine if hardware has an effect on ease of development.

```

Comment:
I think your objectives may have two specific considerations: 
1. What seems to be the minimum necessary hardware configuration for performing development on this open source project?  
2. What software tools does the operating system need to support in order to be effective.